libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionErrorMessage.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.di.ipv.core.library.auditing.extension;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@Getter
+public class AuditExtensionErrorMessage implements AuditExtensions {
```

Comment:
I'm not sure about including the error message here - the audit log is to track what has happened from an audit perspective (so this event says 'we tried and failed to revoke a VC'), and if we include arbitrary exception messages we start having to make sure it doesn't contain anything bad.

I think we probably _should_ include the CRI ID though. This is already a known field type as `extensions.cri` if you want to reuse it.